src/RepoAuditor/Plugins/GitHub/Impl/EnableRulesetRequirementImpl.py

```diff
@@ -59,25 +59,44 @@ def _EvaluateImpl(
         query_data: dict[str, Any],
         requirement_args: dict[str, Any],
     ) -> Requirement.EvaluateImplResult:
+        expected_value = self.enabled_by_default
```

Comment:
I think this is the source of the error I encountered earlier.

IIUC, `enabled_by_default` and `dynamic_arg_name` govern whether we should execute a check or not.
This is different from what we want to check.

Reply:
`enabled_by_default` is related to the actual setting value, but `dynamic_arg_name` governs if we execute the check.